students/jaemoon/users/views.py

```diff
@@ -9,21 +9,21 @@
 class SignupView(View):
     def post(self,request):
         try:
-            data           = json.loads(request.body)
-            email_regex    = r'^[a-zA-Z0-9.-_]+\@[a-zA-Z0-9.-]+\.[a-zA-Z]+$'
-            password_regex = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}'
-            
+            data            = json.loads(request.body)
+            email_regex     = r'^[a-zA-Z0-9.-_]+\@[a-zA-Z0-9.-]+\.[a-zA-Z]+$'
+            password_regex  = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}'
+            hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'),bcrypt.gensalt())
```

Comment:
User를 생성하기 전에 여기서 디코딩을 미리 해주면 되지 않을까요?
```suggestion
            hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'),bcrypt.gensalt()).decode('utf-8')
```